### ISSUE TITLE
Set up workflow between designer and developer

### ISSUE THREAD

**Issue by ethanfrey** [iov-one/wallet-demo#18]
Let's work out a simple workflow based on the wireframes attached to #4 

* Wireframe -> sketch
* sketch -> zepelin? (seems current agreement by Daniel and Andrei)
* zepelin -> react components (story book?)
* demo react components for designer approval

this is really a toy example, but nice place to work out a process and sort out any issues before we have a larger workflow. please add comments here to document/discuss the process as we develop it and we can add it to the README to close the issue.

To recap the call on Tuesday:

- [x] Andrei makes zepelin account and invites both
- [x] Andrei publishes lo-fi design to zepelin
- [x] Daniel reviews zepelin and gives feedback
- [x] Daniel implements design in react
- [x] Andrei reviews app (teach him to run locally)
- [ ] We review flow and adjust (eg. component export)

Idea is to make one end-to-end run and then evaluate how to improve. Will try to integrate with sprints, design "one sprint ahead"
